pom.xml

```diff
@@ -238,21 +238,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
Why did you remove this?

Reply:
I suppressed it because it spanked me whatever error. I just put it back in my last commit: https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa